class/class-mainwp-child-plugins-check.php

```
@@ -344,7 +344,8 @@ public function run_check() { //phpcs:ignore -- NOSONAR - complex.
     private function try_get_response_body( $plugin, $second_pass ) { //phpcs:ignore -- NOSONAR - complex.
 
         // Get the WordPress current version to be polite in the API call.
-        include_once ABSPATH . WPINC . '/version.php'; // NOSONAR - WP compatible.
+        // phpcs:ignore WordPress.NamingConventions.ValidVariableName.UsedPropertyNotSnakeCase -- Using static access for centralized version retrieval
+        $wp_version = MainWP_Child_Server_Information_Base::get_wordpress_version();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$wp_version'. (Unused Code Rules)

(UnusedLocalVariable)